src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -2488,8 +2488,10 @@ public void Bug9288_keyfile()
             var snkPath = snk.Path;
 
             const string source = "";
+            var options = TestOptions.ReleaseDll.WithStrongNameProvider(new DesktopStrongNameProvider()).WithCryptoKeyFile(snkPath);
+            var parseOptions = TestOptions.Regular.WithFeatures(new[] { new KeyValuePair<string, string>("BypassStrongName", "+") });
```

Comment:
Why is `"BypassStrongName"` necessary if we're supplying an explicit `StrongNameProvider`?